src/core_modules/capture-core/components/Pages/MainPage/WorkingLists/epics/eventsQueryArgsBuilder/filterConverters/optionSet/basicDataTypeConverters.js

```diff
@@ -32,7 +33,7 @@ const valueConvertersForType = {
     // $FlowFixMe[prop-missing] automated comment
     [elementTypes.IMAGE]: (rawValue: Object) => rawValue.value,
     // $FlowFixMe[prop-missing] automated comment
-    [elementTypes.COORDINATE]: (rawValue: Object) => `[${rawValue.longitude},${rawValue.latitude}]`,
+    [elementTypes.COORDINATE]: MinimalCoordinates,
```

Comment:
We should revert this change (this is for converting a coordinate filter value to the api)

Reply:
okay didnt know that. Will take care of it. Thanks 🙏